doc/Metrics.md

```diff
@@ -62,12 +64,65 @@ added to the deployment with the following parameters:
   "version": "1.0"
 }
 ```
-After the modules are deployed, Chronograf can be reached at localhost:8888.
+After the modules are deployed, Chronograf can be reached at http://localhost:8888. InfluxDb should be configured as a data source for Chronograf. The InfluxDb instance should be reachable at http://influxdb:8086.
 
 ## Example InfluxDb queries
 
 For example InfluxDb queries, please look at https://github.com/Azure/iotedge/pull/141/files#diff-9fc75ceaff4ae01d42e1bafad8037848
```

Comment:
Since this PR has been merged we could alternatively just link to the source. For example: https://github.com/Azure/iotedge/blob/master/tools/snitch/snitcher/src/settings.yaml#L16

Reply:
Sure. Will update.